### DISCUSSION TITLE
En esta carrera, que tan importante es saber sobre el hardware de las PC en general?

### DISCUSSION THREAD

**Discussion by Pedro6094** [hackatomik/SISCOx256#42] [Chill | Carrera]
Que tan relevante será el estudio de hardware en esta carrera, ya que vemos que kos enfocamos más en la parte de programación y no hemos dado mucho énfasis por el momento q la parte hardware

**Comment by quantumquirkxyz** [comment]
En la carrera de Ingeniería de Sistemas y Computación, entender el hardware de las PC es bastante importante, incluso si el enfoque principal está en la programación. Conocer cómo funciona el hardware ayuda a escribir código más eficiente, resolver problemas de manera más efectiva, y desarrollar software que interactúe bien con los componentes del sistema. Además, es útil para diseñar sistemas, garantizar la interoperabilidad entre diferentes dispositivos, y adaptarse a nuevas tecnologías. Aunque la programación es clave, una buena comprensión del hardware complementa el conocimiento técnico y mejora la capacidad para crear soluciones completas y funcionales.

**Reply by quantumquirkxyz** [reply]
Si no eres de Ing. de Sistemas y Computación, te daría una explicación con un enfoque a tu carrera :D
